parse.go

```diff
@@ -0,0 +1,53 @@
+package vagrantutil
+
+import (
+	"bufio"
+	"strings"
+)
+
+// parseData parses the given vagrant type field from the machine readable
+// output (records).
+func (v *Vagrant) parseData(records [][]string, typeName string) (string, error) {
```

Comment:
I didn't meant this. What I mean is you should use logging after you return from this functions. Example:

```go
result, err := parseData()
if err != nil {
    v.debugf("%s", err)
}
```